### ISSUE TITLE
OpenVidu Browser 2.13 does not support Edge

### ISSUE THREAD

**Issue by tomasz-lasko** [OpenVidu/openvidu#466]
**Describe the bug**
OpenVidu Browser 2.13 (JS client) installed via npm does not support Edge when working with OpenVidu 2.13 server, because either the server or the browser throws a BROWSER_NOT_SUPPORTED error.

**Expected behavior**
When using OpenVidu 2.13 (server and JS client) in Microsoft Edge browser, then webcam streaming etc should work the same as in Chrome and Firefox.

**Wrong current behavior**
BROWSER_NOT_SUPPORTED error

**How to replicate the error**
1. Have openvidu server 2.13 and openvidu browser 2.13 in your JS app (installed via npm, the release version).
2. Start up OpenVidu session in a standard flow (tokens, session, publishing, ...).
3. Observe the following problem:

- nothing happens in Edge (no camera view etc),
- and the javascript console contains these messages:
```
openvidu-server version: 2.13.0

Unhandled promise rejection [object Object]: 
{message: "Browser Microsoft Edge (version 18.18363) for Windows is not supported in OpenVidu", 
name: "BROWSER_NOT_SUPPORTED"}
```

**OpenVidu deployment info**
- We have a OpenVidu server 2.13 run on premise (our own server) in docker - according to the latest official deployment guide docs.
- As a client API we are using OpenVidu Browser 2.13 in javascript app by installing it via: npm install openvidu-browser

**Client device info (if applicable)**
- Microsoft Edge 44.18362.449.0 (Microsoft EdgeHTML 18.18363) on  Windows 10 Pro 64bit (1909 ver 10.0.18363.592)
- but other Edge users reported to me that they also have a problem
- However this problem is NOT seen on ANY recent version of Firefox, Chrome, Vivaldi, only happens on Edge.

**Additional context**
@micaelgallego told me briefly: "It (Edge) should be supported (in the chromium version). If not, please report an issue. " But I do not know what is "**the chromium version**", so I report the issue here.

**Comment by alexlapa** [comment]
> The new Microsoft Edge  is based on Chromium and was released on January 15, 2020.

> verify that you have Microsoft Edge (Chromium) and not Microsoft Edge (EdgeHTML). A quick way to check is to load edge://settings/help in the browser and confirm that the version number is 75 or newer.

**Comment by tomasz-lasko** [comment]
Ah.... now I get it

Because supported page does not specify which Edge version is supported, and release notes tell about "new" Edge which is not clear, I did not catch that.

**So the question is**: what needs to be done, in order to have a proper Edge = supported by OpenVidu 2.13? 

I know it can be out of the scope of OpenVidu, but some short info should be provided. And if docs don't have that info, I kindly ask for someone to put it here and later it can be put also to docs.

**The other question is**: is the old Edge being planned to be supported any way some point on the roadmap?

**In other words**: when my appliaction detects an old version of Edge, can it say to the user something like:
"Please click THIS link to have a new Edge (OR: install THIS update). Otherwise you (dear user) have to wait about a year when we will start supporting the old Edge too."

Thanks!

**Comment by pabloFuente** [comment]
**is the old Edge being planned to be supported any way some point on the roadmap?**
No

**when my appliaction detects an old version of Edge, can it say to the user something like:**

Of course. A short method to check if OpenVidu is supported in a specific platform is to call [OpenVidu.checkSystemRequirements](https://docs.openvidu.io/en/2.13.0/api/openvidu-browser/classes/openvidu.html#checksystemrequirements)

If you wan to know exactly what platform and version Javascript is running on, then you are right, that's out of scope of OpenVidu. But there are a 1000 ways of getting the platform, version, OS, etc in JS. So just do a mix of both: before trying to connect to a session, check that OpenVidu is supported. If not, gather all of the information you want using your favourite method outside of OpenVidu and present it to the user.

**Comment by tomasz-lasko** [comment]
Regarding second question, I know how to detect old Edge, but thanks for the checkSystemRequirements hint.

The question was: what does the user have to do to have the new Edge supported by OpenVidu? is there a single link to download or an particular Windows update to point out? I think this should be definitely mentioned in the docs, at least shortly. So perhaps someone can put it here as well.

So this question could be also rephrased as: can I give (to the end user) just one, two or three simple steps (at a general level, perhaps some links) so that they can update to new Edge themselves? 

Thanks!

**Comment by pabloFuente** [comment]
Of course you can do that. If you detect an old Edge version, just show to the user a warning saying "You must update to the latest Edge version" and if you want a link to the Microsoft download page. https://www.microsoft.com/en-us/edge

**Comment by tomasz-lasko** [comment]
So this link is enough, just one step process? Great, thanks! I hope someone adds  to OpenVidu docs this clear distinction between old and new Edge and puts this single link to update.
